src/ui/components/Collections/Collections.jsx

```diff
@@ -143,7 +143,9 @@ const Collections = props => {
         headers = [
           { value: 'id', label: 'id', edit: false },
           { value: 'fullUrl', label: 'fullUrl', edit: true },
-          { value: 'criteria', label: 'criteria', edit: true },
+          { value: 'criteria', label: 'criteria', edit: false },
+          { value: 'server', label: 'location', edit: false },
+          { value: 'timestamp', label: 'last updated', edit: false },
```

Comment:
same problem with timestamp as above, when you make a new resource this field should be automatically filled out.

Reply:
Also added to PlanDefinition `name`